packages/react/src/components/toast/tests/ToastContainer.spec.tsx

```diff
@@ -100,5 +100,25 @@ describe('Toasts', () => {
 
             expect(() => component.find(Toast).props().onClose()).not.toThrow();
         });
+
+        it('should called the onClose if defined on the toast', () => {
+            let result = '';
+            const newToastAttributes = _.extend({}, basicProps, {
+                toasts: [
+                    {
+                        id: 'toast-id',
+                        title: 'greatest guitarist',
+                        onClose: () => {
+                            result = 'allAlongTheWatchtower';
+                        },
+                    },
+                ],
+            });
+
+            component.setProps(newToastAttributes).mount();
+            component.find(Toast).props().onClose();
+
+            expect(result).toBe('allAlongTheWatchtower');
```

Comment:
Not that I dislike Jimmy Hendrix, but we usually go for spies, when we want to assert something was called.

```suggestion
            const onCloseSpy = jest.fn();
            const newToastAttributes = _.extend({}, basicProps, {
                toasts: [
                    {
                        id: 'toast-id',
                        title: 'greatest guitarist',
                        onClose: onCloseSpy,
                    },
                ],
            });

            component.setProps(newToastAttributes).mount();
            component.find(Toast).props().onClose();

            expect(result).toHaveBeenCalledTimes(1);
```